exporter/containerimage/writer.go

```diff
@@ -333,6 +385,132 @@ func (ic *ImageWriter) commitDistributionManifest(ctx context.Context, ref cache
 	}, &configDesc, nil
 }
 
+func (ic *ImageWriter) commitAttestationsManifest(ctx context.Context, p exptypes.Platform, target string, oci bool, attestations []exporter.Attestation, inTotos []InTotoAttestation) (*ocispecs.Descriptor, *ocispecs.Descriptor, error) {
+	var (
+		attestationType                = exptypes.MediaTypeAttestationType
+		manifestType                   = ocispecs.MediaTypeImageManifest
+		configType                     = ocispecs.MediaTypeImageConfig
+		annotationReferenceType        = exptypes.OCIAnnotationReferenceType
+		annotationReferenceDigest      = exptypes.OCIAnnotationReferenceDigest
+		annotationReferenceDescription = exptypes.OCIAnnotationReferenceDescription
+	)
+	if !oci {
+		attestationType = exptypes.MediaTypeDockerSchema2AttestationType
+		manifestType = images.MediaTypeDockerSchema2Manifest
+		configType = images.MediaTypeDockerSchema2Config
+		annotationReferenceType = exptypes.DockerAnnotationReferenceType
+		annotationReferenceDigest = exptypes.DockerAnnotationReferenceDigest
+		annotationReferenceDescription = exptypes.DockerAnnotationReferenceDescription
+	}
+
+	layers := []ocispecs.Descriptor{}
+	for _, inToto := range inTotos {
```

Comment:
I'd use `for _, statement := range statements` here... "intotos" is kind of a funny word ;)